curation/gene_services.py

```diff
@@ -1,27 +1,26 @@
 """Wrapper for required Gene Normalization services."""
+from curation import logger
 from gene.query import QueryHandler
 from gene.schemas import MatchType
+from typing import List, Tuple, Optional, Dict
+from boto3.dynamodb.conditions import Key
```

Comment:
These aren't used. Wondering why pre-commit isn't picking this up?
```suggestion
from typing import List, Tuple, Optional
```

Reply:
Manually running flake8 shows this. I would try installing pre-commit again and fixing these

Reply:
❔ ❔ ❔